frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -36,7 +42,9 @@ import { ProviderConnection, unpackProviderConnection } from '../../../../../res
 import { Secret } from '../../../../../resources'
 import { createResource as createResourceTool } from '../../../../../resources'
 import { FeatureGates } from '../../../../../FeatureGates'
-import { getNetworkingPatches } from '../components/cim/utils'
+import { getNetworkingPatches } from './components/assisted-installer/utils'
+import { NetworkConfigurationValues } from 'openshift-assisted-ui-lib/dist/src/common/types/clusters'
```

Comment:
We should use imports via the `CIM` namepsace. If needed, please re-export in the ai-ui-lib

Reply:
we can do that in next release of ai-ui-lib

Reply:
fixed